docs/tempo/website/guides/pushing-spans-with-http.md

```diff
@@ -74,7 +74,7 @@ docker run --env BACKEND=localhost:3100 --net host grafana/tempo-query:latest
 
 And open `http://localhost:16686/trace/0123456789abcdef` in the browser of your choice to see:
 
-<p align="center"><img src="pushing-spans-with-http.png" alt="Tempo Logo"></p>
+<p align="center"><img src="pushing-spans-with-http.png" alt="single span"></p>
```

Comment:
When I right-click on the image https://grafana.com/docs/tempo/latest/guides/pushing-spans-with-http/#retrieving-traces here and select "Open in new tab" it takes me to https://grafana.com/docs/tempo/latest/guides/pushing-spans-with-http/pushing-spans-with-http.png

which means its looking for the image in a folder `pushing-spans-with-http`. If I change that link to "../pushing-spans-with-http.png" it works

<img width="1338" alt="Screenshot 2021-02-19 at 1 02 36 PM" src="https://user-images.githubusercontent.com/10982987/108472507-d8ae9f80-72b2-11eb-85dd-622b3108a9fb.png">

Reply:
good catch.  some of our links need the `../` and some don't apparently.  i do not have the time to figure out why :).  `../` added!